question-servers/freeformPythonLib/prairielearn.py

```diff
@@ -341,6 +351,10 @@ def string_from_2darray(A, language='python', presentation_type='f', digits=2):
 
         [ ... ... ; ... ... ]
 
+    If language is 'mathematica' and A is a 2D ndarray, the string looks like this:
+
+        {{ ..., ... },{ ..., ... }}
+
```

Comment:
does the comment below need updating now that 1D arrays are supported?

Reply:
actually, it looks like the comments above need updating as well

Reply:
And the function name should not be `string_from_2darray()` anymore. Maybe `string_from_array()`?